### PR TITLE
Upgrade CAMERA to 1.48

### PR DESCRIPTION
**Pull request by yguitton**
Signed-off-by: yguitton <yann.guitton@gmail.com>

FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by lecorguille** [comment]
For the side packages, can you try:
 - r-snow 0.4_3
 - bioconductor-multtest 2.48.0
 - r-batch 1.1_5
 - graphicsmagick 1.3.26

This combinaison will also pull bioconductor-xcms 3.14.0 where our current xcms is 3.12.0. I hope that it will be armless.

**Comment by yguitton** [comment]
Hello,
 I need some help to regenerate the test-data for the test 1 and test2 of camera_annotate. Planemp --update_test_data is not working for me

Best
Yann

**Comment by bgruening** [comment]
@yguitton you could `planemo serve` run your tool and save the test files.

File: tools/camera_macro/macros.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <xml name="requirements">
         <requirements>
             <requirement type="package" version="0.4_3">r-snow</requirement>
+			<requirement type="package" version="1.2.6">bioconductor-genomeinfodbdata</requirement>
```
Comment (bgruening) [inline tools/camera_macro/macros.xml:8]: indentation is off in this commit, please use 4 spaces instead of a tab

**Comment by lecorguille** [comment]
Green !
`zip` wasn't available within this docker image

**Review by lecorguille — APPROVED** [review]

**Comment by lecorguille** [comment]
Is it ok for you @bgruening?

**Review by bgruening — APPROVED** [review]
Yes, looks great!

**Comment by yguitton** [comment]
Many thanks

**Pull request merged**